### ISSUE TITLE
Relay: OnIncomingMessage not returning a msg body

### ISSUE THREAD

**Issue by sbsends** [signalwire/signalwire-golang#84]
```go
consumer.OnIncomingMessage = func(consumer *signalwire.Consumer, msgObj *signalwire.MsgObj) { 
   msgObj.?????????
}

```

I'm struggling to figure out how to correctly parse out the message body from the incoming message. How do I access the body?

**Comment by dragos-oancea** [comment]
thanks for reporting. it was an oversight. please see: https://github.com/signalwire/signalwire-golang/pull/85

**Comment by dragos-oancea** [comment]
merged #85 , closing this, please  reopen if needed or open other issues if you notice other wrong things.
